cpp-client/deephaven/tests/src/test_util.cc

```diff
@@ -116,11 +149,15 @@ TableMakerForTests TableMakerForTests::Create() {
 }
 
 Client TableMakerForTests::CreateClient(const ClientOptions &options) {
-  const char *hostptr = std::getenv("DH_HOST");
-  const char *portptr = std::getenv("DH_PORT");
-  std::string host = (hostptr == nullptr) ? "localhost" : hostptr;
```

Comment:
It feels like you would have to do this on every test.   Maybe you could add a "get_or_defautl(env_var_name, default_value)" method to the class?

Separate, but maybe related, what's the advantage of using the third argument from main and having to have a static class that can be initialized wrong, as opposed to having some static method that would just call getenv (and maybe provide a default, like the first suggestion in this comment)?

This seems to suggest in windows using the env variable is not a great idea: https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/getenv-wgetenv?view=msvc-170

"getenv operates only on the data structures accessible to the run-time library and not on the environment "segment" created for the process by the operating system. Therefore, programs that use the envp argument to [main](https://learn.microsoft.com/en-us/cpp/cpp/main-function-command-line-args?view=msvc-170) or [wmain](https://learn.microsoft.com/en-us/cpp/cpp/main-function-command-line-args?view=msvc-170) may retrieve invalid information."

I am not sure what that actually menas.  Other than Windows is a mess, that is.